docs/sphinx/sections/api/apidocs/libraries/dagster-dbt.rst

```diff
@@ -17,7 +17,7 @@ dagster-dbt
 ***********
 
 .. click:: dagster_dbt.cli.app:project_app_typer_click_object
-    :prog: dagster-dbt project scaffold
+    :prog: dagster-dbt project
     :nested:
```

Comment:
Let's just add a new item for `dagster-dbt project prepare-for-deployment`.

It doesn't render properly when the full command is omitted: https://dagster-docs-qk9unul9n-elementl.vercel.app/_apidocs/libraries/dagster-dbt#dagster-dbt